app/src/test/java/com/x8bit/bitwarden/ui/platform/feature/vaultunlockednavbar/VaultUnlockedNavBarScreenTest.kt

```diff
@@ -27,10 +27,8 @@ class VaultUnlockedNavBarScreenTest : BaseComposeTest() {
         // When changing root navigation state, pop everything else off the back stack:
         popUpTo(fakeNavHostController.graphId) {
             inclusive = false
-            saveState = true
         }
         launchSingleTop = true
-        restoreState = true
```

Comment:
This concerns me a bit as well. I don't think we want to make this change.

If the problem is with clicking the same bottom nav icon multiple times, can't we just block _that_ from happening or make the nav options different for just those cases?